docs/topics/vscode-extension-reviewing-projects.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-reviewing-projects_{context}']
+= Reviewing issues
+
+The {ProductShortName} extension uses icons to indicate the severity of an issue and wherher or not an issue can be solved by applying a Quick Fix (automatic code replacement).
+
+.Procedure
+
+. Select a run configuration directory on the Run configuration pane and expand its folders to view the *Hints* generated for each application file.
+. Select a hint to view the source code.
+. Right-click a hint and select *View Details* to view the *Rule ID* and other information.
++
+The icon beside the hint displays the issue severity and whether a Quick Fix is available.
++
+The following icons indicate the severity and state of an issue:
++
+.{ProductShortName} extension issue icons
+[cols="15%a,85%",options="header",]
+|====
+|Icon |Description
+|image::vs_mandatory.png[Mandatory] |The issue must be fixed for a successful migration.
+|image::vs_optional.png[Optional or Quick Fix] |The issue is either optional to fix for migration _or_ it can be resolved by applying a Quick Fix.
+
+To determine which meaning is correct, right-click the issue and look at the options in the list that opens. If *Preview Quick Fix* appears, then a Quick Fix is available. Otherwise, fixing it is optional.
```

Comment:
I think you should remove the first clause (which is correct). It does not sound good. Like there's a bug in the UI that we can't be bothered to fix. I still think it is a mistake to use the same icon for two different meanings. You'll have to figure out how to write this so that it does not sound like a bug.